ratings.css

```diff
@@ -0,0 +1,40 @@
+.content {
+	width: 420px;
+	margin-top: 100px
+}
+
+.ratings {
+	background-color: #fff;
+	padding: 5.5rem;
+	border: 1px solid rgba(0, 0, 0, 0.1);
+	box-shadow: 0px 10px 10px #E0E0E0;
```

Comment:
maintain consistency to use the color. you are using both hex code and rgba . it will be better to use vaiable name